app/assets/stylesheets/frontend.less

```diff
@@ -16,34 +16,43 @@ body {
 	font-family: sans-serif;
 }
 
-.section_headings {
-    position: relative;
-    z-index: 1;
-    overflow: hidden;
-    text-align: center;
+h1 {
+	/* Page itself has top padding */
+	margin-top: 0
 }
 
-.section_headings:before, .section_headings:after {
```

Comment:
Aye, sorry about that, well, I'd like to try it that way for now, for the reason I stated below.